README.md

```diff
@@ -16,46 +16,6 @@ The following organization permission should be set:
 
 - Self-hosted runners: Read and write
 
-## Add Continuous Deployment to PROD hosting
-
-Create `.github/workflows/deploy-prod.yml` containing :
-
-```yml
-# https://github.com/EPFL-ENAC/github-actions-runner#readme
-name: deploy-prod
-
-on:
-  push:
-    branches:
-      - main
-
-jobs:
-  deploy:
-    runs-on: [self-hosted, deploy, prod]
-    steps:
-      - uses: EPFL-ENAC/epfl-enac-deploy-action@main
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-```
-
-## Add Continuous Deployment to TEST hosting
-
-Create `.github/workflows/deploy-test.yml` containing :
-
-```yml
-# https://github.com/EPFL-ENAC/github-actions-runner#readme
-name: deploy-test
-
-on:
-  push:
-    branches:
-      - develop
-
-jobs:
-  deploy:
-    runs-on: [self-hosted, deploy, test]
-    steps:
-      - uses: EPFL-ENAC/epfl-enac-deploy-action@main
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-```
+## Known instances
```

Comment:
instances ?!

Reply:
a better word for this ?

Reply:
Continuous Deployment Documentation ?

Reply:
To me, the purpose here is not to link to the documentation.
`github-actions-runner` is a generic project that allows us to implement such runners. I would have found it useful to list here all known instances of it. You see what I mean ?

Regarding the documentation, our internal docs will directly point to the right place, so we don't need to add a link here in my opinion.